extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -127,19 +125,12 @@ public function getWikiVariables() {
 	}
 
 	/**
-	 * @desc Gets a wikia sitename either from the message or WF variable
+	 * @desc Gets a wikia sitename from WF variable
 	 *
 	 * @return null|String
 	 */
 	public function getSiteName() {
-		$siteName = F::app()->wg->Sitename;
-		$msg = wfMessage( static::SITENAME_MSG_KEY )->inContentLanguage();
-
-		if( !$msg->isDisabled() ) {
-			$siteName = $msg->text();
-		}
-
-		return $siteName;
+		return F::app()->wg->Sitename;
```

Comment:
What do you think about moving this to line [122/120](https://github.com/Wikia/app/pull/6697/files#diff-21bab35e67dfc42ffdd18afcf2c13b6cR120)? `getWikiVariables()` already has cached `$wg = F::app()->wg;`.